capstone-rs/src/arch/sysz.rs

```diff
@@ -1,3 +1,106 @@
 //! Contains sysz-specific types
```

Comment:
Please add a `test_arch_sysz_detail()` in `capstone-rs/src/test.rs` (you can see similar tests for other archs).

Reply:
Added this test